code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -245,6 +247,30 @@
 	item_state = "luckpacket"
 	item_obj = /obj/item/clothing/mask/smokable/cigarette/lucky
 
+/obj/item/storage/fancy/cigarettes/robusto
+	name = "\improper Red Robusto packet"
+	desc = "A packet of six Red Robusto Cigarettes. Old NanoTrasen cigs, some say they make you more robust..."
+	icon_state = "robpacket"
+	item_state = "robpacket"
+	spawn_tags = SPAWN_TAG_CIGARETTE_MAINTS
+	item_obj = /obj/item/clothing/mask/smokable/cigarette/robusto
+
+/obj/item/storage/fancy/cigarettes/buckshot
+	name = "\improper Klassika Filter packet"
+	desc = "A packet of six Klassika Filter Cigarettes. Old unknown brand, says 'Klassika Filter' on the front..."
+	icon_state = "bpacket"
+	item_state = "bpacket"
+	spawn_tags = SPAWN_TAG_CIGARETTE_MAINTS
+	item_obj = /obj/item/clothing/mask/smokable/cigarette/buckshot
+
+/obj/item/storage/fancy/cigarettes/quasimorph
+	name = "\improper Kosmos packet"
+	desc = "A packet of six Kosmos Cigarettes. Old Russian brand, says 'Kocmoc' on the front, along with a white rocket."
+	icon_state = "kpacket"
```

Comment:
I know you're just following existing sprite names but Jesus this is badly obfuscated

Reply:
ah. fair. want me to rename it to kosmospacket and klassikapacket?

Reply:
if yes, it's done. if not, unresolve.